### PR TITLE
Train model with similar settings as in original deeplabv3plus paper

### PR DESCRIPTION
**Pull request by MortenTabaka**

### CODE REVIEW

**Comment by MortenTabaka** [comment]
Optimizer: `tensorflow.python.keras.optimizer_v2.adam.Adam` with decaying learning rate (`rate_decay = 0.94`) and weight decay (`decay=4e-5`). 

## SemanticSegmentationLoss.unet3p_hybrid_loss 
deeplabv3plus_v12.0.1
```
    training_set:
      best_loss: 0.28995466232299805
      best_accuracy: 0.9297395348548889
      best_mIoU: 0.693867564201355
    validation_set:
      best_val_loss: 0.5246396064758301
      best_val_accuracy: 0.8697471022605896
      best_val_mIoU: 0.5997530221939087
```

## SemanticSegmentationLoss.basnet_hybrid_loss
Revision `deeplabv3plus_v12.1.2` reached second best results with:
```
     training_set:
      best_loss: 0.13775938749313354
      best_accuracy: 0.9693020582199097
      best_mIoU: 0.8321843147277832
    validation_set:
      best_val_loss: 0.3391178548336029
      best_val_accuracy: 0.9174870848655701
      best_val_mIoU: 0.709352433681488
```
Version 12.1.2 was continuation of training of 12.1.1 with lower initial learning rate and less layers frozen.

12.1.1:
* LAST_LAYER_TO_FREEZE = 359
* initial_learning_rate = 0.005

12.1.2:
* LAST_LAYER_TO_FREEZE = 319
* initial_learning_rate = 0.0001

+-+--------------+
|  |   IoU score|
+-+--------------+
|0|   0.864240 |
|1|  0.631640 |
|2|  0.832095 |
|3|  0.768978 |
|4|  0.443896 |
+-+-------------+



Even though high miou was reached, model has issues with `water` class. 
image
